Sources/NIOIMAPCore/Parser/Grammar/GrammarParser+Fetch.swift

```diff
@@ -271,6 +278,7 @@ extension GrammarParser {
         return try PL.parseOneOf([
             parseFetchResponse_streamingBegin,
             parseFetchResponse_streamingBeginQuoted,
+            parseFetchResponse_nilBody,
             parseFetchResponse_simpleAttribute,
```

Comment:
Instead of adding `parseFetchResponse_nilBody` here, can we handle this case inside `parseMessageAttribute`?

That way `parseMessageAttribute` parses all the possible `MessageAttribute` values.

Reply:
Makes sense. That’s a bit more convoluted to do since `parseMessageAttribute()` uses `parseFromLookupTable()`…